hw/ip/entropy_src/doc/_index.md

```diff
@@ -10,82 +10,95 @@ This module conforms to the [Comportable guideline for peripheral functionality.
 
 ## Features
 
-- This revision supports a LFSR (linear feedback shift register) digital source, and provides an interface to an external rng (random noise generator) source.
-The LFSR is used as a digital, pseudo-random type of entropy source, while the rng external source is a true entropy source.
+- This revision provides an interface to an external RNG (random noise generator) source, and a LFSR (linear feedback shift register) digital source. 
+The LFSR is used as a digital, pseudo-random type of entropy source, while the RNG external source is a digital noise source.
 A noise source and its relation to an entropy source are defined by [SP 800-90B](https://csrc.nist.gov/publications/detail/sp/800-90b/final).
-- A set of registers is provided for firmware to manage and obtain entropy bits.
-- The set of registers is designed such that firmware can select between a digital or an rng entropy source.
+- A set of registers is provided for firmware to obtain entropy bits.
+- The set of registers is designed such that firmware can select between an LFSR or an RNG digital noise source.
 - Interrupts are supported:
   - Entropy bits are available for firmware consumption.
-  - The external rng entropy source is available.
-  - The internal health tests have detected a repetitive count test fail.
-  - The internal health tests have detected an adaptive proportion test fail.
+  - The internal health tests have detected a test fail.
```

Comment:
fail -> failure (? - maybe, I'm not sure if "fail" is NIST-specific jargon here.)

Reply:
Agree.  Thanks @cdgori.